src/restful/controllers/authController.js

```diff
@@ -257,6 +269,7 @@ class AuthController {
     }
   }
 
+
   static async updateUser(req, res) {
     try {
       const { first_name, last_name, bio } = req.body;
```

Comment:
We anticipate that the user profile picture will be processed here.

Reply:
Yes! i get it now,